inference-engine/src/mkldnn_plugin/nodes/mkldnn_eltwise_node.cpp

```diff
@@ -1404,6 +1423,10 @@ void MKLDNNEltwiseNode::createPrimitive() {
 
     if (eltwise_kernel)
         eltwise_kernel->create_ker();
+
+    for (auto i = 0; i < inputNum; i++)
+        memPtrs.push_back(getParentEdgeAt(i)->getMemoryPtr());
+    memPtrs.push_back(getChildEdgeAt(0)->getMemoryPtr());
```

Comment:
Do we really need to move this initialization on LoadNetwork stage? What's the perf impact?